### PR TITLE
Fix assignments saving on role removing

### LINKED ISSUE #10101: Rbac\PhpManager: Missing method call "saveAssignments" in method "removeItem"

### PR DESCRIPTION
**Pull request by rezident**
Fix #10101

### CODE REVIEW

File: framework/CHANGELOG.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ Yii Framework 2 Change Log
 - Chg #9411: `DetailView` now automatically sets container tag ID in case it's not specified (samdark)
 - Chg #9953: `TimestampBehavior::getValue()` changed to make value processing consistent with `AttributeBehavior::getValue()` (silverfire)
 - New #10083: Added wrapper for PHP webserver (samdark)
+- Bug #10101: Fixed assignments saving on role removing (rezident1307)
```
Comment (SilverFire) [inline framework/CHANGELOG.md]: We usually order the changelog by type and issue id.
Could you move this line after the line for `Bug #10029`

**Comment by SilverFire** [comment]
It's fine now, thank you!

@rezident1307 your commits are not linked to your GitHub account. It seems you have missconfigured git on you local. Try to fix it with:

```
git config --global user.email "YOUR EMAIL ADDRESS"
```

All your new commits will be lined to your account

**Comment by rezident1307** [comment]
Thank you! I fix my git global config to future

**Comment by samdark** [comment]
@rezident1307 I can correct commit author before merge. Just need to know your username and email.

**Comment by rezident1307** [comment]
username: rezident1307
email: mail@rezident.org

**Comment by samdark** [comment]
Merged https://github.com/yiisoft/yii2/commit/be407391ce0d9d02d5579ad08d60ea609b5efeac. Thanks!

**Pull request closed without merging**